src/server/scripts/Outland/BlackTemple/boss_illidan.cpp

```diff
@@ -1335,7 +1335,7 @@ struct npc_flame_of_azzinoth : public ScriptedAI
             DoCastVictim(SPELL_FLAME_BLAST);
 
             me->m_Events.AddEventAtOffset([&] {
-                DoCastVictim(SPELL_BLAZE);
+                me->GetVictim()->CastSpell(me->GetVictim(), SPELL_BLAZE, true);
```

Comment:
We need to check victim here, it can be null

Reply:
Corrected.